### ISSUE TITLE
Add a template for node18

### ISSUE THREAD

**Issue by radicand** [openfaas/templates#296]
## Expected Behaviour
Today there are templates for various NodeJS versions. Node 18 is out, but there is no template for it. Node 18 is particularly interesting because it bundles fetch() natively, reducing the need for dependencies within functions.

## Current Behaviour
Use a current NodeJS image, but with a shim/package to add fetch support.

## Are you a GitHub Sponsor (Yes/No?)
No

Check at: https://github.com/sponsors/openfaas
- [ ] Yes
- [x] No

## List All Possible Solutions and Workarounds
Create a new template for Node18
Continue to shim/add dependencies to functions

## Which Solution Do You Recommend?
Creating a new template for Node18

## Steps to Reproduce (for bugs)
N/A

## Context
I'm writing functions that interact with http-based backends primarily and am looking to keep them as slim as possible.

Thanks for considering this!

## Your Environment
N/A

**Comment by maiermic** [comment]
I assume that this issue can be closed, since there is a template for Node.js 18 in the meantime: https://github.com/openfaas/templates/tree/master/template/node18

**Comment by sebastianricaldoni** [comment]
Agreed @maiermic ! Seems this can be closed. There's actually a Node20 template already (check [here](https://github.com/openfaas/templates/tree/master))
